Review.swift

```diff
@@ -0,0 +1,294 @@
+                        logger.info("Inline Comments: \(inlineComments.count)")
+                        
+                        logger.info("Posting Inline Comments...")
+                        for inlineComment in inlineComments {
+                            let comment = Comment(id: nil,
+                                                  line: inlineComment.0.location.line!,
+                                                  lineType: inlineComment.1.type,
+                                                  ruleDescription: inlineComment.0.ruleName,
+                                                  content: inlineComment.0.reason,
+                                                  path: String(inlineComment.0.location.relativeFile?.split(separator: "/", maxSplits: 1, omittingEmptySubsequences: true)[1] ?? ""),
+                                                  type: CommentType(rawValue: inlineComment.0.severity.rawValue))
+                            
+                            sourceControlService.postComment(repositoryName: repository.name, pullRequest: pullRequest, comment: comment, diff: diff, req: req)
+                        }
+                        
+                        var comments: [Comment] = []
+                        
+                        logger.info("Aggregating General Comments...")
+                        for generalComment in generalComments {
+                            comments.append(Comment(id: nil,
+                                                    line: nil,
+                                                    lineType: "",
+                                                    ruleDescription: generalComment.ruleDescription,
+                                                    content: generalComment.reason,
+                                                    path: "",
+                                                    type: CommentType(rawValue: generalComment.severity.rawValue)))
+                        }
+                        
+                        if !comments.isEmpty {
+                            logger.info("Posting General Comments...")
+                            sourceControlService.postGeneralComment(repositoryName: repository.name, pullRequest: pullRequest, comments: comments, req: req)
+                        }
+                        
+                        if generalComments.count == 0 && inlineComments.count == 0 {
+                            logger.info("Approving Pull Request...")
+                            sourceControlService.approvePullRequest(repositoryName: repository.name, pullRequestId: pullRequest.scmId, req: req)
+                        } else {
+                            logger.info("Pull Request Needs Work...")
+                            sourceControlService.markNeedsRework(repositoryName: repository.name, pullRequestId: pullRequest.scmId, req: req).whenSuccess { _ in
```

Comment:
🚫 Error
 [Line Length] At line 103: Line should be 150 characters or less: currently 160 characters